infra/scripts/build-tcm.sh

```diff
@@ -18,7 +18,7 @@ echo ${PROJECT_DIR:=${PWD}}
 
 java -jar $PROJECT_DIR/tca-standalone-0.0.8.jar \
   --outdir=$PROJECT_DIR/tcm-output \
-  --config=$PROJECT_DIR/.ahub/tcchecker-tca/config.yaml \
+  --config=$PROJECT_DIR/src/.ahub/tcchecker-tca/config.yaml \
```

Comment:
There are two `config.yaml` 
- .ahub/tcchecker-tca/config.yaml
- compiler/.ahub/tcchecker-tca/config.yaml

I could not see `src` in the path. What `src` is this for?

Reply:
Found it... -_-;

```
 Create symbolic link `./src` for source directory
```